example/src/screens/calendarsList.js

```diff
@@ -1,12 +1,10 @@
 import React, {Component} from 'react';
 import {CalendarList} from 'react-native-calendars';
 
-
 export default class CalendarsList extends Component {
-  
   render() {
     return (
-      <CalendarList current={'2012-05-16'} pastScrollRange={24} futureScrollRange={24}/>
+      <CalendarList onHeaderPress={date => console.log('Header Pressed!', date)} current={'2012-05-16'} pastScrollRange={24} futureScrollRange={24}/>
```

Comment:
please add an example for the `onHeaderLongPress` as well

Reply:
I would actually move this example to the Calendars example screen so it will not appear as a CalendarList feature only